srv/GetEntities.srv

```diff
@@ -0,0 +1,17 @@
+# Get objects in the scene which can be interacted, e.g. with using SetEntityState.
+
+string filter                                       # Optional, defaults to empty. Return entities with matching names.
+                                                    # Entity names are matched with the filter regular expression.
+                                                    # An empty filter will result in all entities being returned.
+                                                    # Applies together with other filters (categories, tags).
+uint8[] categories                                  # Optional, defaults to empty, which means no category filter.
+                                                    # Entities matching any of the categories will be returned.
+                                                    # To get entity category, use GetEntityInfo.
+                                                    # Applies together with other filters (filter, tags).
+TagsFilter tags                                     # Tags filter to apply. To get entity tags, use GetEntityInfo.
+                                                    # Applies together with other filters (filter, categories).
+
+---
+
+Result result
+EntityWithState[] entities                          # All entities with their states, matching the filters.
```

Comment:
Should this really return the state of all entities??

Often it is prefereable to make multiple small requests rather than a large one (e.g. for entity in GetEntities: GetEntityState(entity) ) .


This is linked to [my comment on having an `Entity` msg](https://github.com/ros-simulation/simulation_interfaces/pull/1/files#r1945499577)

Reply:
I am addressing this:
- GetEntities() (basic)
- GetEntitiesStates() (convenience, performance)
will both be there